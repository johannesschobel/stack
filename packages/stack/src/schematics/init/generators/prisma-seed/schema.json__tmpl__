{
  "cli": "nx",
  "id": "prisma-seed",
  "type": "object",
  "properties": {
    "confirm": {
      "type": "boolean",
      "description": "Library confirm",
      "x-prompt": "Are you sure you want to delete and seed the database?"
    }
  },
  "required": ["confirm"]
}
