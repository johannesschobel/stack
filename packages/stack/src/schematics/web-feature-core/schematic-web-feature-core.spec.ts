import { Tree } from '@angular-devkit/schematics'
import { SchematicTestRunner } from '@angular-devkit/schematics/testing'
import { createEmptyWorkspace } from '@nrwl/workspace/testing'
import { join } from 'path'

import { WebFeatureCoreSchematicSchema } from './schema'

describe('web-feature-core schematic', () => {
  let appTree: Tree
  const options: WebFeatureCoreSchematicSchema = { appName: 'test' }

  const testRunner = new SchematicTestRunner('@nxpm/web-feature-core', join(__dirname, '../../../collection.json'))

  beforeEach(() => {
    appTree = createEmptyWorkspace(Tree.empty())
  })

  it('should run successfully', async () => {
    await expect(
      testRunner.runSchematicAsync('web-feature-core', options, appTree).toPromise(),
    ).resolves.not.toThrowError()

    expect(appTree.read('libs/test/core/feature/src/lib/test-core-feature.module.ts').toString()).toMatchSnapshot()
  })
})
