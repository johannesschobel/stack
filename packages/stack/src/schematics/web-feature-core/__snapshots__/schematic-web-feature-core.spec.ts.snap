// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web-feature-core schematic should run successfully 1`] = `
"import { NgModule } from '@angular/core'
import { HttpClientModule } from '@angular/common/http'

import { TestCoreFeatureGraphQLModule  } from './test-core-feature-graphql.module'

@NgModule({
  imports: [HttpClientModule, TestCoreFeatureGraphQLModule],
})
export class TestCoreFeatureModule {}
"
`;
